lib/bootstrap/2.disk.coffee

```diff
@@ -26,42 +24,43 @@ module.exports = header: "Disk", handler: ({options}) ->
       #{partition_1} : start=     2048, size=   2097152, type=C12A7328-F81F-11D2-BA4B-00A0C93EC93B
       #{partition_2} : start=  2099200, type=E6D6D379-F507-44C2-A23C-238F2A3DF928
       """
-    @system.execute """
+    @execute
+    command: """
     sfdisk #{options.disk} < /root/layout.sfdisk
     """
   @call
-    header: "Partitions Formating"
+    metadata: header: "Partitions Formating"
   , ->
     for partition, info of options.partitions
-      @system.execute switch info.type
+      @execute switch info.type
         when 'f32'
           "mkfs.vfat -F32 -nESP #{partition}"
         when 'ext4'
           "mkfs.ext4 #{partition}"
         else throw Error "Invalid partition type"
   @call
-    header: "Partitions LVM"
+    metadata: header: "Partitions LVM"
   , ->
-    @system.execute
-      header: 'Crypsetup'
-      cmd: """
+    @execute
+      metadata: header: 'Crypsetup'
+      command: """
       echo '#{options.crypt.passphrase}' | \
         cryptsetup luksFormat #{options.crypt.device}
       echo '#{options.crypt.passphrase}' | \
         cryptsetup open --type luks #{options.crypt.device} lvm
       """
       code_skipped: 3
-    @system.execute
-      header: 'Format'
-      cmd: """
+    @execute
+      metadata: header: 'Format'
+      command: """
       info=#{options.info or ''}
       pvcreate /dev/mapper/lvm
       vgcreate volume /dev/mapper/lvm
-      lvcreate -L 4G volume -n swap
-      lvcreate -L 100G volume -n root
+      lvcreate -L 2G volume -n swap
+      lvcreate -L 18G volume -n root
       # VirtualBox sizing
-      # lvcreate -L 2G volume -n swap
-      # lvcreate -L 18G volume -n root
+      lvcreate -L 2G volume -n swap
+      lvcreate -L 18G volume -n root
```

Comment:
don't commit you personal information

Reply:
I will change this and the previous one.